src/components/table/table-head.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './table.css';
+
+export default function TableHead({ children, className, cssRow }) {
+  return (
+    <thead className={className}>
+      <tr className={cssRow}>
+        {React.Children.map(children, (child) => {
+          return React.cloneElement(child, { header: true });
+        })}
```

Comment:
Using `cloneElement` to apply the `header` prop telling TableCell to make it a th instead of a td.

Alternatively, we could have different components for normal cells vs header cells.